src/index.html

```diff
@@ -9,6 +9,24 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+  <header class="header">
+    <div class="header-container">
+      <div class="header-logo">
+        <a href="https://www.moyo.ua/"><img src="images/logo.png" alt="logo"></a>
+      </div>
+      <nav class="header-navbar">
+        <ul class="header-navbar__list">
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link is-active">APPLE</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">SAMSUNG</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a></li>
+          <li class="header-navbar__item" data-qa="hover"><a href="#" class="header-navbar__link">НОУТБУКИ И КОМПЬЮТЕРЫ</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">ГАДЖЕТЫ</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">ПЛАНШЕТЫ</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">ФОТО</a></li>
+          <li class="header-navbar__item"><a href="#" class="header-navbar__link">ВИДЕО</a></li>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.